### ISSUE TITLE
failed to insert record into tables

### ISSUE THREAD

**Issue by hillbun** [ByConity/ByConity#503]
CREATE TABLE test.lcxxx (b LowCardinality(String)) engine=CnchMergeTree ORDER BY b settings storage_policy = 'cnch_default_hdfs';

INSERT INTO test.lcxxx SELECT '0123456789' FROM numbers(100000000);

it return errors:

Code: 8051. DB::Exception: Received from 1.2.3.4:9000. DB::Exception: Received from 10.0.1.39:9000. DB::Exception: DB::Exception: No available topology for current commit time : 1689752919875. Available topology : [] SQLSTATE: HY000.: While executing SinkToOutputStream SQLSTATE: HY000.

**Comment by dmthuc** [comment]
Hi @hillbun , could you check if the server readiness probe are ok?

**Comment by hillbun** [comment]
yes

byconity-server-0                                            1/1     Running     0               50m

**Comment by dmthuc** [comment]
> yes
> 
> byconity-server-0 1/1 Running 0 50m

is this issue always happens or just intermittent? I wonder if all pod connect to the same FDB server successfully?
If it is always happens, could you try to execute `set send_logs_level='trace'` and execute the insert query again and send us the output

**Comment by hillbun** [comment]
it always heppens

```
10.0.2.34 :) INSERT INTO test.my_first_table (user_id, message, timestamp) VALUES
:-]     (101, 'Hello, ByConity!',                                 now()),
:-]     (102, 'Insert a lot of rows per batch',                     yesterday()),
:-]     (102, 'Sort your data based on your commonly-used queries', today()),
:-]     (101, 'Granules are the smallest chunks of data read',      now() + 5);

INSERT INTO test.my_first_table (user_id, message, timestamp) VALUES

Query id: 7ed2a11a-722f-4025-851b-f274c1ca61e9

[byconity-server-0] 2023.07.19 08:35:30.362237 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> TransactionCoordinator: Created txn txn_id: 442959326177918976 commit_ts: 0 status: Running priority: high location: "10.0.2.34:8124" initiator: "Server" type: Implicit

[byconity-server-0] 2023.07.19 08:35:30.362305 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> trySetVirtualWarehouse: Trying to set virtual warehouse for query `INSERT INTO test.my_first_table (user_id, message, timestamp) VALUES` ...
[byconity-server-0] 2023.07.19 08:35:30.362330 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> trySetVirtualWarehouse: Trying to set virtual warehouse for expression `INSERT INTO test.my_first_table (user_id, message, timestamp) VALUES` ...
[byconity-server-0] 2023.07.19 08:35:30.362364 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> trySetVirtualWarehouse: Try set virtual warehouse for table `test.my_first_table`
[byconity-server-0] 2023.07.19 08:35:30.364425 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> DatabaseCnch (test): Create database test in query 
[byconity-server-0] 2023.07.19 08:35:30.368610 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> StorageFactory:  engine name CnchMergeTree
[byconity-server-0] 2023.07.19 08:35:30.369110 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Information> CnchTopologyMaster: No available topology for current commit time : 1689755730354. Available topology : []
[byconity-server-0] 2023.07.19 08:35:30.369451 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> trySetVirtualWarehouse: Set virtual warehouse vw_write from test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973)
[byconity-server-0] 2023.07.19 08:35:30.369821 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> trySetVirtualWarehouse: Picked worker group vw_write.vw_write
[byconity-server-0] 2023.07.19 08:35:30.369900 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> executeQuery: (from [::1]:51438) INSERT INTO test.my_first_table (user_id, message, timestamp) VALUES 
[byconity-server-0] 2023.07.19 08:35:30.370154 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> ContextAccess (default): Access granted: INSERT(user_id, message, timestamp) ON test.my_first_table
[byconity-server-0] 2023.07.19 08:35:30.370263 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): dump with 1 threads
[byconity-server-0] 2023.07.19 08:35:30.371635 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): Start to write new block
[byconity-server-0] 2023.07.19 08:35:30.371715 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): size of part_blocks 4
[byconity-server-0] 2023.07.19 08:35:30.371735 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): size of bucketed_part_blocks 1
[byconity-server-0] 2023.07.19 08:35:30.372066 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> DiskLocal: Reserving 1.00 MiB on disk `server_local_0`, having unreserved 48.71 GiB.
[byconity-server-0] 2023.07.19 08:35:30.372123 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973) (Writer): Writing temp part to auxility_store/9b3c02d1-616c-4719-a8fb-4c5be957c973/tmp_insert_1689755730_442959326177918978_442959326177918978_0_442959326177918976/...

[byconity-server-0] 2023.07.19 08:35:30.372603 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): Write part 1689755730_442959326177918978_442959326177918978_0_442959326177918976, 1 rows, elapsed 0 ms
[byconity-server-0] 2023.07.19 08:35:30.372649 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): size of bucketed_part_blocks 1
[byconity-server-0] 2023.07.19 08:35:30.372936 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> DiskLocal: Reserving 1.00 MiB on disk `server_local_0`, having unreserved 48.71 GiB.
[byconity-server-0] 2023.07.19 08:35:30.372972 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973) (Writer): Writing temp part to auxility_store/9b3c02d1-616c-4719-a8fb-4c5be957c973/tmp_insert_1689638400_442959326177918979_442959326177918979_0_442959326177918976/...

[byconity-server-0] 2023.07.19 08:35:30.377601 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): Write part 1689638400_442959326177918979_442959326177918979_0_442959326177918976, 1 rows, elapsed 4 ms
[byconity-server-0] 2023.07.19 08:35:30.377678 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): size of bucketed_part_blocks 1
[byconity-server-0] 2023.07.19 08:35:30.378007 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> DiskLocal: Reserving 1.00 MiB on disk `server_local_0`, having unreserved 48.71 GiB.
[byconity-server-0] 2023.07.19 08:35:30.378071 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973) (Writer): Writing temp part to auxility_store/9b3c02d1-616c-4719-a8fb-4c5be957c973/tmp_insert_1689724800_442959326177918980_442959326177918980_0_442959326177918976/...

[byconity-server-0] 2023.07.19 08:35:30.378471 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): Write part 1689724800_442959326177918980_442959326177918980_0_442959326177918976, 1 rows, elapsed 0 ms
[byconity-server-0] 2023.07.19 08:35:30.378513 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): size of bucketed_part_blocks 1
[byconity-server-0] 2023.07.19 08:35:30.378834 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> DiskLocal: Reserving 1.00 MiB on disk `server_local_0`, having unreserved 48.71 GiB.
[byconity-server-0] 2023.07.19 08:35:30.378892 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973) (Writer): Writing temp part to auxility_store/9b3c02d1-616c-4719-a8fb-4c5be957c973/tmp_insert_1689755735_442959326177918981_442959326177918981_0_442959326177918976/...

[byconity-server-0] 2023.07.19 08:35:30.379230 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): Write part 1689755735_442959326177918981_442959326177918981_0_442959326177918976, 1 rows, elapsed 0 ms
[byconity-server-0] 2023.07.19 08:35:30.379286 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): Finish converting block into parts, elapsed 7 ms
[byconity-server-0] 2023.07.19 08:35:30.379300 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): Start dump and commit 4 parts, 0 bitmaps, 0 staged parts
[byconity-server-0] 2023.07.19 08:35:30.383670 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): Wrote undo buffer for 4 resources in 4 ms
[byconity-server-0] 2023.07.19 08:35:30.512483 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): Dumped 0 bitmaps (0 not inline) in 0 ms
[byconity-server-0] 2023.07.19 08:35:30.512576 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973): Dumped 4 parts, 0 bitmaps, 0 staged parts in 128 ms
[byconity-server-0] 2023.07.19 08:35:30.512680 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> ThreadPool: clear thread for finalize : [794, 797, 1194, 1193, ]
[byconity-server-0] 2023.07.19 08:35:30.512760 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> Catalog: Start write 4 parts and 0 delete_bitmaps and 0 staged parts to bytekv for txn 442959326177918976
[byconity-server-0] 2023.07.19 08:35:30.513111 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Information> CnchTopologyMaster: No available topology for current commit time : 1689755730503. Available topology : []
[byconity-server-0] 2023.07.19 08:35:30.514266 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> TransactionCoordinator: Deleted txn 442959326177918976

[byconity-server-0] 2023.07.19 08:35:30.514330 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> TransactionCoordinator: No need to erase active timestamp for txn 442959326177918976

[byconity-server-0] 2023.07.19 08:35:30.514373 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> CnchServerTransaction: Start abort transaction 442959326177918976

[byconity-server-0] 2023.07.19 08:35:30.519712 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> CnchServerTransaction: Successfully abort transaction: 442959326177918976

[byconity-server-0] 2023.07.19 08:35:30.519759 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> CnchServerTransaction: Start clean transaction: 442959326177918976

[byconity-server-0] 2023.07.19 08:35:30.519816 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Information> Catalog: Start clear metadata of 4 parts, 0 delete bitmaps, 0 staged parts of table test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973).
[byconity-server-0] 2023.07.19 08:35:30.524059 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Information> Catalog: Finish clear metadata of 4 parts, 0 delete bitmaps, 0 staged parts of table test.my_first_table (9b3c02d1-616c-4719-a8fb-4c5be957c973).
[byconity-server-0] 2023.07.19 08:35:30.528105 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> StorageFactory:  engine name CnchMergeTree
[byconity-server-0] 2023.07.19 08:35:30.528386 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> StoragePolicy::getDiskByName: Storage disk name: server_hdfs_disk loaded
[byconity-server-0] 2023.07.19 08:35:30.528433 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion DefaultConfig::ctor takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.528550 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: FsImpl Connect to <hdfs>://<10.1.2.8>:9000
[byconity-server-0] 2023.07.19 08:35:30.528604 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion hdfsBuilderConnect takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.528655 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFSFileSystem: Reconnect host: hdfs://10.1.2.8
[byconity-server-0] 2023.07.19 08:35:30.529437 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion hdfsExists takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.529477 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> StoragePolicy::getDiskByName: Storage disk name: server_hdfs_disk loaded
[byconity-server-0] 2023.07.19 08:35:30.529531 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion DefaultConfig::ctor takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.529634 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: FsImpl Connect to <hdfs>://<10.1.2.8>:9000
[byconity-server-0] 2023.07.19 08:35:30.529686 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion hdfsBuilderConnect takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.529733 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFSFileSystem: Reconnect host: hdfs://10.1.2.8
[byconity-server-0] 2023.07.19 08:35:30.530465 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion hdfsExists takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.530512 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> StoragePolicy::getDiskByName: Storage disk name: server_hdfs_disk loaded
[byconity-server-0] 2023.07.19 08:35:30.530556 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion DefaultConfig::ctor takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.530658 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: FsImpl Connect to <hdfs>://<10.1.2.8>:9000
[byconity-server-0] 2023.07.19 08:35:30.530708 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion hdfsBuilderConnect takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.530751 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFSFileSystem: Reconnect host: hdfs://10.1.2.8
[byconity-server-0] 2023.07.19 08:35:30.531432 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion hdfsExists takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.531484 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> StoragePolicy::getDiskByName: Storage disk name: server_hdfs_disk loaded
[byconity-server-0] 2023.07.19 08:35:30.531529 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion DefaultConfig::ctor takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.531638 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: FsImpl Connect to <hdfs>://<10.1.2.8>:9000
[byconity-server-0] 2023.07.19 08:35:30.531672 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion hdfsBuilderConnect takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.531722 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFSFileSystem: Reconnect host: hdfs://10.1.2.8
[byconity-server-0] 2023.07.19 08:35:30.532398 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Trace> HDFS3: Hdfs opertaion hdfsExists takes 0 ms
[byconity-server-0] 2023.07.19 08:35:30.540950 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Debug> CnchServerTransaction: Successfully clean a failed transaction: 442959326177918976

[byconity-server-0] 2023.07.19 08:35:30.541132 [ 228 ] {7ed2a11a-722f-4025-851b-f274c1ca61e9} <Error> executeQuery: Code: 8051, e.displayText() = DB::Exception: No available topology for current commit time : 1689755730503. Available topology : [] SQLSTATE: HY000 (version 21.8.7.1) (from [::1]:51438) (in query: INSERT INTO test.my_first_table (user_id, message, timestamp) VALUES ), Stack trace (when copying this message, always include the lines below):

0. Poco::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int) @ 0x19e02db2 in /opt/byconity/bin/clickhouse
1. DB::Exception::Exception(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, int, bool) @ 0xb665400 in /opt/byconity/bin/clickhouse
2. DB::CnchTopologyMaster::getTargetServerImpl(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::list<DB::CnchServerTopology, std::__1::allocator<DB::CnchServerTopology> >&, unsigned long, bool, bool) @ 0x13aafee2 in /opt/byconity/bin/clickhouse
3. DB::CnchTopologyMaster::getTargetServer(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, bool, bool) @ 0x13ab0458 in /opt/byconity/bin/clickhouse
4. DB::Catalog::Catalog::assertLocalServerThrowIfNot(std::__1::shared_ptr<DB::IStorage> const&) const @ 0x13acc843 in /opt/byconity/bin/clickhouse
5. DB::Catalog::Catalog::finishCommitInBatch(std::__1::shared_ptr<DB::IStorage> const&, DB::TxnTimestamp const&, DB::Protos::DataModelPartVector const&, std::__1::vector<std::__1::shared_ptr<DB::DeleteBitmapMeta>, std::__1::allocator<std::__1::shared_ptr<DB::DeleteBitmapMeta> > > const&, DB::Protos::DataModelPartVector const&, bool, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&) @ 0x13acce1d in /opt/byconity/bin/clickhouse
6. ? @ 0x13aeca7a in /opt/byconity/bin/clickhouse
7. DB::Catalog::Catalog::writeParts(std::__1::shared_ptr<DB::IStorage> const&, DB::TxnTimestamp const&, DB::Catalog::CommitItems const&, bool, bool) @ 0x13ad04ee in /opt/byconity/bin/clickhouse
8. DB::InsertAction::executeV2() @ 0x13b8b28a in /opt/byconity/bin/clickhouse
9. DB::CnchDataWriter::commitPreparedCnchParts(DB::DumpedData const&) @ 0x13b6d640 in /opt/byconity/bin/clickhouse
10. DB::CnchDataWriter::commitDumpedParts(DB::DumpedData const&) @ 0x13b6c51d in /opt/byconity/bin/clickhouse
11. DB::CnchDataWriter::dumpAndCommitCnchParts(std::__1::vector<std::__1::shared_ptr<DB::IMergeTreeDataPart>, std::__1::allocator<std::__1::shared_ptr<DB::IMergeTreeDataPart> > > const&, std::__1::vector<std::__1::shared_ptr<DB::LocalDeleteBitmap>, std::__1::allocator<std::__1::shared_ptr<DB::LocalDeleteBitmap> > > const&, std::__1::vector<std::__1::shared_ptr<DB::IMergeTreeDataPart>, std::__1::allocator<std::__1::shared_ptr<DB::IMergeTreeDataPart> > > const&) @ 0x13b69b7e in /opt/byconity/bin/clickhouse
12. DB::CnchDataWriter::schedule(std::__1::vector<std::__1::shared_ptr<DB::IMergeTreeDataPart>, std::__1::allocator<std::__1::shared_ptr<DB::IMergeTreeDataPart> > > const&, std::__1::vector<std::__1::shared_ptr<DB::LocalDeleteBitmap>, std::__1::allocator<std::__1::shared_ptr<DB::LocalDeleteBitmap> > > const&, std::__1::vector<std::__1::shared_ptr<DB::IMergeTreeDataPart>, std::__1::allocator<std::__1::shared_ptr<DB::IMergeTreeDataPart> > > const&) @ 0x13b6ea57 in /opt/byconity/bin/clickhouse
13. DB::CloudMergeTreeBlockOutputStream::write(DB::Block const&) @ 0x14ff72a7 in /opt/byconity/bin/clickhouse
14. DB::PushingToViewsBlockOutputStream::write(DB::Block const&) @ 0x1472ce12 in /opt/byconity/bin/clickhouse
15. DB::AddingDefaultBlockOutputStream::write(DB::Block const&) @ 0x1472aa4c in /opt/byconity/bin/clickhouse
16. DB::SquashingBlockOutputStream::finalize() @ 0x1472a194 in /opt/byconity/bin/clickhouse
17. DB::SquashingBlockOutputStream::writeSuffix() @ 0x1472a24e in /opt/byconity/bin/clickhouse
18. DB::CountingBlockOutputStream::writeSuffix() @ 0x13fffb23 in /opt/byconity/bin/clickhouse
19. DB::TCPHandler::processInsertQuery(DB::Settings const&) @ 0x1535a1af in /opt/byconity/bin/clickhouse
20. DB::TCPHandler::runImpl() @ 0x15355078 in /opt/byconity/bin/clickhouse
21. DB::TCPHandler::run() @ 0x1536241c in /opt/byconity/bin/clickhouse
22. Poco::Net::TCPServerConnection::start() @ 0x19d7e9cc in /opt/byconity/bin/clickhouse
23. Poco::Net::TCPServerDispatcher::run() @ 0x19d7eeac in /opt/byconity/bin/clickhouse
24. Poco::PooledThread::run() @ 0x19e5fb5a in /opt/byconity/bin/clickhouse
25. Poco::ThreadImpl::runnableEntry(void*) @ 0x19e5d70c in /opt/byconity/bin/clickhouse
26. start_thread @ 0x7fa3 in /lib/x86_64-linux-gnu/libpthread-2.28.so
27. clone @ 0xf906f in /lib/x86_64-linux-gnu/libc-2.28.so


4 rows in set. Elapsed: 0.185 sec. 

Received exception from server (version 21.8.7):
Code: 8051. DB::Exception: Received from localhost:9000. DB::Exception: No available topology for current commit time : 1689755730503. Available topology : [] SQLSTATE: HY000. 
```
